modules/hcp-ibmz-update-reg-ca.adoc

```diff
@@ -0,0 +1,56 @@
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-update-registry-ca-private-registry_{context}"]
+= Update Registry Certificate Authority in AgentServiceConfig with Mirror Registry
+
+When using a mirror registry for images, agents need to trust the registry's Certificate to securely pull images. For this, we must add the Certificate Authority of mirror registry to `AgentServiceConfig` by creating a `ConfigMap`.
+
+. After the installation of MultiCluster Engine (MCE) operator, create 'ConfigMap' with the mirror registry details under the same namespace where MCE is installed. This ensures that you grant the hosted cluster workers the capability to retrieve images from the mirror registry.
++
+.Example ConfigMap file
+[source,terminal]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mirror-config
+  namespace: multicluster-engine
+  labels:
+    app: assisted-service
+data:
+  ca-bundle.crt: |
+    -----BEGIN CERTIFICATE-----
+    -----END CERTIFICATE-----
+  registries.conf: |
+
+    [[registry]]
+      location = "registry.stage.redhat.io"
+      insecure = false
+      blocked = false
+      mirror-by-digest-only = true
+      prefix = ""
+
+      [[registry.mirror]]
+        location = "<mirror_registry>"
+        insecure = false
+
+    [[registry]]
+      location = "registry.redhat.io/multicluster-engine"
+      insecure = false
+      blocked = false
+      mirror-by-digest-only = true
+      prefix = ""
+
+      [[registry.mirror]]
+        location = "<mirror_registry>/multicluster-engine"
```

Comment:
If the user needs to replace any variables in this code, be sure to include callouts to explain what needs to be replaced.